### DISCUSSION TITLE
more examples

### DISCUSSION THREAD

**Discussion by Ying-Zi66** [jlbs86/mpi4py_CuTonala_2024_A_Parallel_Programming#44] [General]
your examples areexcellent！everyone can learn easily mpi4py from your examples。i think the examples are maybe fundamental。there are not some  complex examples。could you add some advanced examples for practical problems？I hope  I can learn more skills about mpi4py。for example， run on multiple hosts。

**Comment by jlbs86** [comment]
Sure, think!! I finished all my easy-to-complex examples working in single and multiple nodes.
